### DISCUSSION TITLE
Jetson AGX Ori: how to select network interface used by ROS 2?

### DISCUSSION THREAD

**Discussion by valdivj** [iRobotEducation/create3_docs#95] [Questions]
I was able to get My Jetson AGX Orin to connect to the IRobot create3 and execute ROS2 commands. 
But I notice that the AGX Orin seems to be sending the commands to the IRobot create3 over WIFI.

I noticed this because with the cable disconnected from the Jetson AGX Orin to the IRobot create3 the IRobot create3  is still able to  execute commands sent from the Jetson AGX Orin.

 Also the LOG file shows the USB connection from the Jetson AGX Orin to the IRobot create3 is connected when the cable is plugged in and shows the connection is broken when the cable is unplugged.

My question is how or were do I look to see how the Jetson AGX Orin has decided what communication protocol  ROS2 uses to talk to the  IRobot create3

**Comment by justinIRBT** [comment]
I don't know enough about DDS layer internals to know where to look for communication ports / network traffic, but you have some control over the DDS layer through config files.  There is some documentation about config files here:
https://iroboteducation.github.io/create3_docs/setup/xml-config/
If you want to ensure your communication is going over a usb0 interface for example, you could configure your DDS to specifically use that interface.  More information about config options can be found in documentation for the respective DDS layers
